cli/ioctl/cmd/account/accountcreateadd.go

```diff
@@ -39,7 +39,12 @@ func accountCreateAdd(args []string) (string, error) {
 	}
 	alias := args[0]
 	if addr, ok := config.ReadConfig.Aliases[alias]; ok {
-		return "", fmt.Errorf("alias \"%s\" has already used for %s", alias, addr)
+		var confirm string
+		fmt.Printf("** Alias \"%s\" has already used for %s\nType 'YES' to continue, quit for anything else.\n", alias, addr)
```

Comment:
Can you check if it overwrites the previous keystore file, or keep as it is?

Reply:
wallet keeps the previous,but alias binds a new key.Should delete the previous or keep it?